utils/data_manager/modules/routecalc.py

```diff
@@ -106,16 +106,16 @@ def calculate_new_route(self, coords, max_radius, max_coords_within_radius, dele
                             route_name: str = 'Unknown'):
         if overwrite_calculation:
             calc_type = 'quick'
-        self.recalc_status = 1
+        self.set_recalc_status(True)
         if in_memory is False:
             if delete_old_route:
                 self.new_calc = True
                 logger.debug("Deleting routefile...")
                 self._data['fields']['routefile'] = []
-        self.save()
+                self.save()
         new_route = self.getJsonRoute(coords, max_radius, max_coords_within_radius, in_memory, num_processes=num_procs,
                                       algorithm=calc_type, useS2=useS2, S2level=S2level, route_name=route_name)
-        self.recalc_status = 0
+        self.set_recalc_status(False)
         self.save()

```

Comment:
Why is not this save removed in mem mode?

Reply:
Must have missed the delete :\  fixed in 65b41c5